File: typescript/src/examples/evm/createAccount.ts
// Usage: pnpm tsx src/examples/evm/createAccount.ts

import { config } from "dotenv";

import { CdpClient } from "../../index";

/**
 * This example shows how to create a new EVM account
 */
async function main() {
  config();

  const cdp = new CdpClient();
  const account = await cdp.evm.createAccount();
  console.log("EVM Account Address: ", account.address);
}

main().catch(console.error);

File: python/cdp/examples/evm/create_account.py
# Usage: uv run python cdp/examples/evm/create_account.py

import asyncio
from dotenv import load_dotenv

from cdp import CdpClient


async def main():
    load_dotenv()

    async with CdpClient() as cdp:
        evm_account = await cdp.evm.create_account()
        print(f"Successfully created EVM account: {evm_account.address}")


# Run the async function
asyncio.run(main())
